setup.cfg

```
@@ -22,6 +22,7 @@ install_requires =
     numpy
     scipy
     pyyaml
+    skypy-data @ https://github.com/skypyproject/data/archive/master.tar.gz
```

```suggestion
    skypy-data @ https://github.com/skypyproject/skypy-data/archive/master.tar.gz
```
Small detail: if we are going to start distributing a package called skypy-data can we update the repository name to match the package name?